src/main/sensors/barometer.c

```diff
@@ -348,16 +348,16 @@ static void baroSetCalibrationCycles(uint16_t calibrationCyclesRequired)
 
 void baroStartCalibration(void)
 {
-    baroSetCalibrationCycles(NUM_CALIBRATION_CYCLES);
-    baroGroundAltitude = 0;
     baroCalibrated = false;
+    baroGroundAltitude = 0;
+    baroSetCalibrationCycles(NUM_CALIBRATION_CYCLES);
```

Comment:
Why has the order been changed here? I don’t see how this will make any difference. The call simply sets `calibrationCycles`.

Reply:
Reverted. See below.